extensions/interactions/SetInput/directives/oppia-interactive-set-input.component.ts

```diff
@@ -79,6 +79,9 @@ export class InteractiveSetInputComponent implements OnInit {
   }
 
   updateAnswer(answer: SetInputAnswer): void {
+    const hasDuplicates = this.hasDuplicates(answer);
+    this.errorMessage = hasDuplicates ?
+    'Oops, it looks like your answer has duplicates!' : '';
```

Comment:
The error message needs to be internationalized. Please fix. See https://github.com/oppia/oppia/wiki/How-to-develop-for-i18n#adding-new-translation-keys for more info.

Reply:
> The error message needs to be internationalized. Please fix. See https://github.com/oppia/oppia/wiki/How-to-develop-for-i18n#adding-new-translation-keys for more info.

It has been updated now, Thanks!